### PR TITLE
Fixed neko head.

### PR DESCRIPTION
**Pull request by andyli**
Neko now uses CMake to build.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Nice! Any idea when there will be a stable release using CMake?

**Comment by andyli** [comment]
Hopefully soon, but we have been [talking about a new release](https://github.com/HaxeFoundation/neko/issues/69) for nearly a year...

File: Formula/neko.rb [hunk 1]
```diff
@@ -16,29 +16,35 @@ class Neko < Formula
     sha256 "a45ce3f4eab713bea15f8b34045333462d3e6a971c10257b9789ffc8000951e2" => :mountain_lion
   end
 
+  depends_on "cmake" => :build if build.head?
```
Comment (mikemcquaid) [inline Formula/neko.rb]: Put these all in a `head do` block.

Comment (andyli) [inline Formula/neko.rb]: Done : )

**Comment by mikemcquaid** [comment]
Thanks @andyli!

**Pull request merged**